### DISCUSSION TITLE
Show event highlights for unselectable days in QDate

### DISCUSSION THREAD

**Discussion by ryhn** [quasarframework/quasar#13886] [Ideas / Proposals]
when set options in qdate event highlight not appear for them. It would be nice if the props could be added like ":always-highlight"

**Comment by metalsadman** [comment]
you can check in your options function if the date exist in your `events` array. https://codepen.io/metalsadman/pen/KKoprra

**Comment by ryhn** [comment]
I guess I didn't explain well. I want event indicator to be visible on unselectable dates.

**Reply by metalsadman** [reply]
what is an `unselectable dates`?

**Reply by metalsadman** [reply]
https://github.com/quasarframework/quasar/pull/13910
